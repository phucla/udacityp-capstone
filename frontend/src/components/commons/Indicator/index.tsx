const Indicator = () => (
  <div className="indicator">
    <span className="indicator__img" />
  </div>
)

export default Indicator
