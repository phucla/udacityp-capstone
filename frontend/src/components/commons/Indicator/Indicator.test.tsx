// Libs
import React from 'react'
import { shallow, ShallowWrapper } from 'enzyme'

// Components
import Indicator from './index'

describe('Indicator component', () => {
  let wrapper: ShallowWrapper

  beforeEach(() => {
    wrapper = shallow(<Indicator />)
  })

  // Test Snapshots
  it('should render correctly UI', () => {
    expect(wrapper).toMatchSnapshot()
  })
})
