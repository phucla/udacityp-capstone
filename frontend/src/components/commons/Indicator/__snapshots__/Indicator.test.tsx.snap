// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Indicator component should render correctly UI 1`] = `
<div
  className="indicator"
>
  <span
    className="indicator__img"
  />
</div>
`;
